.ruby-version

```diff
@@ -0,0 +1 @@
+2.2.0
```

Comment:
Do we need this file? I have `.ruby-version` git ignored on my machine. I also have it equal to `2.1.5@git-town` currently for the gemset. If we are adding this can we throw on `@git-town`